### ISSUE TITLE
Addon: Default contacts on new user creation

### ISSUE THREAD

**Issue by hoergen** [friendica/friendica#6000]
There should be an addon where you can define some default contacts and forums, when a new user is created or a new user has registered. 

Eventually one could extend the "suggest contact" feature with a set of default contacts, that will be triggered automatically, when a new user is created one or the other way.

For example : 

- Node Admin
- local forums
- new here / support forum

**Comment by tobiasd** [comment]
> There should be an addon where you can define some default contacts
> and forums, when a new user is created or a new user has registered. 

Sounds like an expansion (and eventual rename) of the "New Member Widget" addon.

**Comment by tobiasd** [comment]
Yes and no. I think that your suggestion sounds like something that should go into this addon. And then addon should be renamed. Otherwise admins of public nodes that want to have such extra functionality would need to install more then one addon with almost identical purposes (help new users to find support).
